src/redux/greeting/greeting_slice.js

```diff
@@ -0,0 +1,26 @@
+// eslint-disable-next-line import/no-extraneous-dependencies
+import { createSlice, createAsyncThunk } from '@reduxjs/toolkit';
+
+const initialState = [];
+
```

Comment:
As requested by the previous code reviewer please kindly enable eslint in all files were it is disabled.